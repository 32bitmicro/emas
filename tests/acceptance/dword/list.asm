.dword 70000, 80000, 90000, -70000, -80000, -90000
