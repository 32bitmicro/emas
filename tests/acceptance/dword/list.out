@ 0x0000 : 0x0001  /  000 000 0 000 000 001  /  1
@ 0x0001 : 0x1170  /  000 100 0 101 110 000  /  4464
@ 0x0002 : 0x0001  /  000 000 0 000 000 001  /  1
@ 0x0003 : 0x3880  /  001 110 0 010 000 000  /  14464
@ 0x0004 : 0x0001  /  000 000 0 000 000 001  /  1
@ 0x0005 : 0x5f90  /  010 111 1 110 010 000  /  24464
@ 0x0006 : 0xfffe  /  111 111 1 111 111 110  /  65534
@ 0x0007 : 0xee90  /  111 011 1 010 010 000  /  61072
@ 0x0008 : 0xfffe  /  111 111 1 111 111 110  /  65534
@ 0x0009 : 0xc780  /  110 001 1 110 000 000  /  51072
@ 0x000a : 0xfffe  /  111 111 1 111 111 110  /  65534
@ 0x000b : 0xa070  /  101 000 0 001 110 000  /  41072
